tests/unit/pages/subjects/Subjects.spec.jsx

```diff
@@ -60,18 +60,6 @@ describe('Subjects', () => {
     ).toBeInTheDocument()
   })
 
-  it('should update search value when search input is changed', () => {
```

Comment:
> why did you remove this test case if this test is not passing after your changes, maybe it needs to be modified rather than deleted.

returned the test back